text/0000-grid-primitives.md

```diff
@@ -0,0 +1,605 @@
+- Feature Name: Primitives
+- Start Date: TBD
+- RFC PR: TBD
+
+# Summary
+[summary]: #summary
+
+Grid Primitives for Hyperledger Grid provides a reusable, standard approach to
```

Comment:
We should call out what the name of this smart contract will be called.

Reply:
The current suggestion for this is `grid_schema`. Are there any alternatives?

Reply:
Added.